driver/src/main/java/org/neo4j/driver/internal/messaging/v3/BoltProtocolV3.java

```diff
@@ -189,4 +213,8 @@ public BoltProtocolVersion version() {
     protected boolean includeDateTimeUtcPatchInHello() {
         return false;
     }
+
+    protected boolean includeNotificationSettings() {
+        return false;
+    }
```

Comment:
The notification information should not be swallowed by the driver. If the protocol doesn't support send notification filters and error should be thrown.

Reply:
Updated as per the discussion.